Simplified/Audiobooks/Bookmark/NYPLAudiobookBookmarksBusinessLogic.swift

```diff
@@ -5,17 +5,43 @@
 //  Created by Ernest Fan on 2022-08-12.
 //  Copyright © 2022 NYPL. All rights reserved.
 //
-
+#if FEATURE_AUDIOBOOKS
 import Foundation
 import NYPLAudiobookToolkit
 
 class NYPLAudiobookBookmarksBusinessLogic: NYPLAudiobookBookmarksBusinessLogicDelegate {
```

Comment:
The logic of all the functions in this class is basically identical to `NYPLReadiumBookmarksBusinessLogic`, I made some minor changes so it works with `NYPLAudiobookBookmark`.

Reply:
I think they are gonna stay pretty close. And yes, the changes are just syntactic because of different types. The reason I don't want to make it generic is because 
1. The logic is already complicated and I don't want to add any code on top of that
2. This class access book registry very often and if we use generic we need to check the class of the current object before every call of the book registry, unless we add another layer of protocol

Reply:
sounds good..... is most of the duplication concentrated in NYPLBookRegistry? that duplication is somewhat necessary because it is mainly rerouting to the NYPLAudiobookToolkit apis.

Reply:
Created a ticket [iOS-447](https://jira.nypl.org/browse/IOS-447) for this issue.